app/data/data-chat-read-timestamp/data-chat-read-timestamp-public/src/main/kotlin/com/hedvig/android/data/chat/read/timestamp/ChatLastMessageReadRepository.kt

```diff
@@ -0,0 +1,54 @@
+package com.hedvig.android.data.chat.read.timestamp
+
+import arrow.core.toNonEmptyListOrNull
+import com.apollographql.apollo3.ApolloClient
+import com.apollographql.apollo3.cache.normalized.FetchPolicy
+import com.apollographql.apollo3.cache.normalized.fetchPolicy
+import com.hedvig.android.apollo.safeExecute
+import com.hedvig.android.apollo.toEither
+import kotlinx.datetime.Instant
+import octopus.ChatLatestMessageTimestampsQuery
+
+interface ChatLastMessageReadRepository {
```

Comment:
`LastChatMessageReadRepository`?

Reply:
Yeah I'll go with that! I think I confused myself with chat being first but then the name was so forgetable